hibernate-core/src/main/java/org/hibernate/query/sqm/function/SelfRenderingSqmAggregateFunction.java

```diff
@@ -47,10 +48,10 @@ public SelfRenderingFunctionSqlAstExpression convertToSqlAst(SqmToSqlAstConverte
 		return new SelfRenderingAggregateFunctionSqlAstExpression(
 				getFunctionName(),
 				getRenderingSupport(),
-				resolveSqlAstArguments( getArguments(), walker ),
+				resolveSqlAstArguments( getArguments(), (BaseSqmToSqlAstConverter) walker ),
```

Comment:
Seems unnecessary, or am I missing something?

Reply:
you are right, going to remove the cast. Thanks